stories/object-attribute/object-attribute.stories.js

```diff
@@ -0,0 +1,33 @@
+export default {
+    title: 'Dev/Object Attribute',
+    components: ['object-attribute']
+};
+
+export const Text = () => `
+
+<div class="fd-object-attribute" id="standardtext" aria-label="Standard Object Attribute Text">Standard Object Attribute Text</div>
+`;
+
+Text.parameters = { docs: { disable: true } };
+
+
+export const TextWithLink = () => `
+<div id="firsttext" class="fd-object-attribute" aria-label="Object Attribute">Object Attribute :
+<span id="textwithlink" class="fd-object-attribute--link" aria-label="Text With Link" role="link" tabindex="0">Text With Link</span></div>
+`;
+
+TextWithLink.parameters = { docs: { disable: true } };
+
+export const DisabledLink = () => `
+    <span id="disabledlink" class="fd-object-attribute fd-object-attribute--link" role="link" tabindex="0" disabled="true" aria-disabled="true" aria-label="Disabled Object Attribute linked Text">Disabled Object Attribute linked Text</span>
+`;
+
+DisabledLink.parameters = { docs: { disable: true } };
+
+export const TruncatedText = () => `
+<div style="width: 120px;">
+    <span id="truncated" class="fd-object-attribute" tabindex="0" title="Truncate Very long Object Attribute Text" aria-label="Truncate Very long Object Attribute Text">Truncate Very long Object Attribute Text</span>
```

Comment:
`aria-label` is redundant if the text is fully in the DOM; screen readers can see the full text even if it's truncated for sighted users.
```suggestion
    <span id="truncated" class="fd-object-attribute" tabindex="0" title="Truncate Very long Object Attribute Text">Truncate Very long Object Attribute Text</span>
```